### ISSUE TITLE
In 1.5.2-198 unknown length videos no longer filtered by Short Videos : Hidden

### ISSUE THREAD

**Issue by 2600box** [yattee/yattee#850]
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/yattee/yattee/issues) and I haven't found bug report like this

### Current Behavior

Unknown length videos are no longer filtered by `Short videos: hidden` in version 1.5.2-198

### Expected Behavior

Short videos and videos of unknown length that are always short anyway, should not appear in my feed

### Device & OS Version

iOS 18.1

### App Build

1.5.2-598

### App Settings

Short videos: hidden

### Screenshots, Videos and other files

![IMG_2627](https://github.com/user-attachments/assets/0c651c83-2dfe-4174-8cc4-467f6dbd9654)

**Comment by derspyy** [comment]
wdym by 'videos of unknown length that are always short anyway'?

**Comment by 2600box** [comment]
> wdym by 'videos of unknown length that are always short anyway'?

Not sure what is needed, but all these new videos that now appear in my feed since 198, that do not have a time listed, are less than sixty seconds long, and should be able to be filtered one way or another.

Many of them have #shorts in the title...

Some examples in case that helps?

https://www.youtube.com/watch?v=Pkbhn8sIWuw
https://www.youtube.com/watch?v=zOinEmmUUdk
https://www.youtube.com/watch?v=k8VahB3PM7k 
https://www.youtube.com/watch?v=lkWRiks6luc

**Comment by derspyy** [comment]
are you on inv.nadeko.net?

if so, we can't tell shorts and regular videos apart through time because we aren't receiving video length anymore.

**Comment by 2600box** [comment]
> are you on inv.nadeko.net?
> 
> if so, we can't tell shorts and regular videos apart through time because we aren't receiving video length anymore.

No, I don't use any public instances. I have my own invidious instance that I use. Normal videos still show the runtime on latest releases in my feed, only the shorts do not:

![IMG_2638](https://github.com/user-attachments/assets/ba53fa0f-8615-4a88-a7d6-2c1b6137e772)

**Comment by derspyy** [comment]
can u plz show me how it looks on the browser?

**Comment by 2600box** [comment]
This is my invidious instance in firefox on desktop:

![Screenshot 2024-12-28 at 12 49 34](https://github.com/user-attachments/assets/68957d1b-aca1-4cb9-b24f-12c737442283)

**Comment by derspyy** [comment]
i feel like this is a invidious issue.

<img width="197" alt="image" src="https://github.com/user-attachments/assets/9a26e629-f8f0-4594-9ffd-7f372d1870e5" />

these should have a known length. 

does this only happen on subscriptions page?

**Comment by 2600box** [comment]
I had to scroll for a while on the trending page to find anything, and it was a video with its length reported as 35 seconds.

![image](https://github.com/user-attachments/assets/33a20256-ca04-43f5-b9b8-2797a8d21054)

When I look at the api reported for the LTT short, it has the length there and is reported as a short in the api url. So it seems to be working as expected there?

 `lengthSeconds":60`

https://invidious.inv.inst/api/v1/channels/UCXuqSBlHAE6Xw-yeJA0Tunw/shorts

```
{"videos":[{"type":"video","title":"I Built A Secret Gaming Room","videoId":"6uyI6s4XJLw","author":"Linus Tech Tips","authorId":"UCXuqSBlHAE6Xw-yeJA0Tunw","authorUrl":"/channel/UCXuqSBlHAE6Xw-yeJA0Tunw","authorVerified":false,"videoThumbnails":[{"quality":"maxres","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/maxres.jpg","width":1280,"height":720},{"quality":"maxresdefault","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/maxresdefault.jpg","width":1280,"height":720},{"quality":"sddefault","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/sddefault.jpg","width":640,"height":480},{"quality":"high","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/hqdefault.jpg","width":480,"height":360},{"quality":"medium","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/mqdefault.jpg","width":320,"height":180},{"quality":"default","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/default.jpg","width":120,"height":90},{"quality":"start","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/1.jpg","width":120,"height":90},{"quality":"middle","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/2.jpg","width":120,"height":90},{"quality":"end","url":"https://invidious.inv.inst/vi/6uyI6s4XJLw/3.jpg","width":120,"height":90}],"description":"","descriptionHtml":"","viewCount":64000,"viewCountText":"64K views","published":0,"publishedText":"55 years ago","lengthSeconds":60,"liveNow":false,"premium":false,"isUpcoming":true,"premiereTimestamp":0,"isNew":false,"is4k":false,"is8k":false,"isVr180":false,"isVr360":false,"is3d":false,"hasCaptions":false}
```
